src/include/sof/audio/kpb.h

```diff
@@ -15,8 +15,16 @@
 struct comp_buffer;
 
 /* KPB internal defines */
+
+#ifdef CONFIG_TIGERLAKE
 #define KPB_MAX_BUFF_TIME 3000 /**< time of buffering in miliseconds */
 #define HOST_WAKEUP_TIME 1000 /* aprox. time of host DMA wakup from suspend [ms] */
+#else
+/** Due to memory constraints on non-TGL platforms, the buffers are smaller. */
+#define KPB_MAX_BUFF_TIME 2100 /**< time of buffering in miliseconds */
+#define HOST_WAKEUP_TIME 0 /* aprox. time of host DMA wakup from suspend [ms] */
+#endif
+
```

Comment:
@slawblauciak @mrajwa @keyonjie @lgirdwood 

question on "ultimately the buffer length and wakeup time settings should be moved to the topology"

Sure, but do we have an IPC for this? And is this tracked anywhere as a requested feature?

Reply:
Not yet. It was mostly an internal discussion.

Reply:
We could still use the existed config blob, but updating its structure.

Reply:
@slawblauciak can you create a FEATURE for KPB buffer in topology. We can add it in v1.7. Flow will be KPB will use platform defaults unless overridden by topology.

I will merge this a v1.6 fix unless any objections ?

Reply:
yes, that is the ultimate solution.